### DISCUSSION TITLE
groundwater model

### DISCUSSION THREAD

**Discussion by Joaquinfnz** [mhm-ufz/mHM#72] [Q&A]
"Hello, I need help coupling the OGS model to MHM. My question is, does the output of OGS get reintroduced to MHM? In this way, what are the outputs that I should consider? Thank you very much."

**Comment by rohinikumar82** [comment]
The coupling of mHM with OGS is one-way, meaning that the mHM-simulated GW recharge is taken as input to the OGS model.

**Reply by Joaquinfnz** [reply]
I understand, do you know if there is any way for OGS to give us the header levels?

and my last question is,
Do you know any alternative that gives us, a bi-directional model?

**Reply by rohinikumar82** [reply]
I guess OGS can give the head-level information (double-check this with the OGS community - I am not upto the mark with OGS development).. Sorry, I do not know about a bi-directional model.
